### PR TITLE
Pass features from cpanfile to optional_features in META.json

### LINKED ISSUE #35: FR: convert features in cpanfile to optional_features in META
ref: https://github.com/miyagawa/Dist-Zilla-Plugin-Prereqs-FromCPANfile/commit/304965810a0fd43a813e6e742fea0f7f50b62cfc

### PR DESCRIPTION
**Pull request by mavit**
Fixes #35.

### CODE REVIEW

File: lib/Minilla/Project.pm [hunk 1]
```diff
@@ -455,6 +455,7 @@ sub cpan_meta {
         version        => $self->version,
         name           => $self->dist_name,
         prereqs        => $merged_prereqs,
+        optional_features => {},
```
Comment (skaji) [inline lib/Minilla/Project.pm]: This makes META files always have optional_features field.
I don't think we need it if it is empty.

Comment (mavit) [reply lib/Minilla/Project.pm]: I considered this.  It turns out that if it is empty, it doesn't get written into `META.json` (at least with the version of CPAN::Meta I tested with, 2.150010).  However, I can rewrite it to add this key only if non-empty, if you prefer?

Comment (skaji) [reply lib/Minilla/Project.pm]:
```
❯ perl -MCPAN::Meta\ 99999
CPAN::Meta version 99999 required--this is only version 2.150010.
BEGIN failed--compilation aborted.

❯ perl -Ilib script/minil dist --no-test

❯ tar xf Minilla-v3.1.0.tar.gz

❯ grep optional_features Minilla-v3.1.0/META.*
Minilla-v3.1.0/META.json:   "optional_features" : {},
Minilla-v3.1.0/META.yml:optional_features: {}
```

Comment (mavit) [reply lib/Minilla/Project.pm]: Apologies.  I was looking only at the `META.json` created in the top-level directory, and had not realised that this file was generated differently for the tarball.

**Comment by skaji** [comment]
LGTM

**Comment by skaji** [comment]
@mavit Thanks. Released https://metacpan.org/release/SKAJI/Minilla-v3.1.1

**Pull request merged**